src/state/routing/slice.ts

```diff
@@ -91,7 +91,7 @@ export const routing = createApi({
               tokenOutAddress,
               tokenOutChainId,
               amount,
-              tradeType,
+              type: tradeType === TradeType.EXACT_INPUT ? 'exactIn' : 'exactOut',
```

Comment:
nit: should `exactIn` and `exactOut` be enum values?

Reply:
These are the values expected by the `quote` API.
The API should maybe allow our existing tradeType enum values, but it makes sense that these are strings.